.changeset/tricky-adults-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": minor
```

Comment:
Could also be considered a patch change, I wasn't sure. But doesn't really matter